test/e2e/framework/framework.go

```diff
@@ -26,15 +30,24 @@ type Framework struct {
 
 var Global *Framework
 
+var (
+	domain        = zms.DomainName("k8s.omega.stage.kube-test")
```

Comment:
Can we make the domain, trust domain, and test domain command line arguments into the e2e framework?